release_notes/v1.3.md

```diff
@@ -0,0 +1,342 @@
+- <b>Haystack Integration</b>: Enabled OPEA as a backend of [Haystack](https://haystack.deepset.ai). ([Haystack-OPEA#1](https://github.com/opea-project/Haystack-OPEA#1))
+- <b>Cloud Readiness</b>: Expanded automated Terraform deployment for ChatQnA to include support for Azure, and enabled CodeGen deployments on AWS and GCP. ([GenAIExamples#1731](https://github.com/opea-project/GenAIExamples/pull/1731))
+
+### New GenAI Capabilities
+- <b>OPEA Store</b>: Delivered a unified data store access API and a robust data store integration layer that streamlines data store integration. ArangoDB was ported supporting data preparation and retrieval. ([GenAIComps#1528](https://github.com/opea-project/GenAIComps/issues/1528))
+- <b>CodeGen using RAG and Agent</b>: Leveraged RAG and code agent to provide an additional layer of intelligence and adaptability for CodeGen example. ([GenAIExamples#1757](https://github.com/opea-project/GenAIExamples/pull/1757))
+- <b>Enhanced Multimodality</b>: Added support for additional audio file types (.mp3) and supported spoken audio captions with image ingestion. ([GenAIExamples#1549](https://github.com/opea-project/GenAIExamples/issues/1549))
+- <b>Struct to Graph</b>: Supported transforming structured data to graphs using Neo4j graph database. ([GenAIComps#1502](https://github.com/opea-project/GenAIComps/pull/1502))
+- <b>Text to Graph</b>: Supported creating graphs from text by extracting graph triplets. ([GenAIComps#1357](https://github.com/opea-project/GenAIComps/pull/1357), [GenAIComps#1472](https://github.com/opea-project/GenAIComps/pull/1472))
+- <b>Text to Cypher</b>: Supported generating and executing Cypher queries from natural language for graph database retrieval. ([GenAIComps#1319](https://github.com/opea-project/GenAIComps/pull/1319))
+
+### Enhanced Evaluation
+- <b>Enhanced Long-Context Model Evaluation</b>: Supported evaluating long-context model on Intel® Gaudi® with vLLM. ([HELMET#20](https://github.com/princeton-nlp/HELMET/pull/20))
+- <b>TAG-Bench for SQL Agents</b>: Integrated [TAG-Bench](https://github.com/TAG-Research/TAG-Bench) to evaluate complex SQL query generation ([GenAIEval#230](https://github.com/opea-project/GenAIEval#230)).
+- <b>DocSum Support</b>: GenAIEval now supports evaluating the performance of DocSum. ([GenAIEval#252](https://github.com/opea-project/GenAIEval/pull/252))
+- <b>Toxicity Detection Evaluation</b>: Introduced a workflow to evaluate the capability of detecting toxic language based on LLMs. ([GenAIEval#241](https://github.com/opea-project/GenAIEval/pull/241))
+- <b>Model Card</b>: Added a model card generator for generating reports containing model performance and fairness metrics. ([GenAIEval#236](https://github.com/opea-project/GenAIEval/pull/236))
+
+### Observability
+- <b>OpenTelemetry Tracing</b>: Leveraged OpenTelemetry to enable tracing for ChatQnA and AgentQnA on TGI and TEI. ([GenAIExamples#1542](https://github.com/opea-project/GenAIExamples/issues/1542))
```

Comment:
We have OPEA specific tracing. TGI and TEI tracing are from huggingface.  They are no dependency among them, so better to use along with instead of on. 
```suggestion
- <b>OpenTelemetry Tracing</b>: Leveraged OpenTelemetry to enable tracing for ChatQnA and AgentQnA along with TGI and TEI. ([GenAIExamples#1542](https://github.com/opea-project/GenAIExamples/issues/1542))
```